rancher-2.9.json

```diff
@@ -38,18 +52,19 @@
       "allowedVersions": "<0.31.0"
     },
     {
+      "enabled": true,
       "description": "Disable major bumps",
       "matchPackageNames": [
         "rancher/kuberlr-kubectl"
       ],
       "matchUpdateTypes": ["minor", "patch"]
     },
     {
+      "enabled": false,
       "groupName": "GitHub Actions",
       "matchManagers": [
         "github-actions"
-      ],
-      "enabled": false
+      ]
     }
```

Comment:
Now that we are disabling everything by default, we could just remove the GitHub Actions section here.